### ISSUE TITLE
Error while getting audio/video decrytption keys

### ISSUE THREAD

**Issue by pragyach** [OpenSTDL/CrunchyDL#65]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by stratuma** [comment]
Are you sure you use my downloader, because I already include the binaries.

**Comment by pragyach** [comment]
yes I cloned your project and tried getting widewine keys using android emulator but unable to
